kong/db/strategies/postgres/connector.lua

```diff
@@ -338,21 +339,26 @@ function _mt:init_worker(strategies)
         return
       end
 
-      local ok, _, _, num_queries = self:query(cleanup_statement)
+      local ok, err, _, num_queries = self:query(cleanup_statement)
       if not ok then
-        for i = num_queries + 1, cleanup_statements_count do
-          local statement = cleanup_statements[i]
-          local ok, err = self:query(statement)
-          if not ok then
-            if err then
-              log(WARN, "unable to clean expired rows from table '",
-                        sorted_strategies[i], "' on postgres database (",
-                        err, ")")
-            else
-              log(WARN, "unable to clean expired rows from table '",
-                        sorted_strategies[i], "' on postgres database")
+        if num_queries then
+          for i = num_queries + 1, cleanup_statements_count do
+            local statement = cleanup_statements[i]
+            local ok, err = self:query(statement)
+            if not ok then
+              if err then
+                log(WARN, "unable to clean expired rows from table '",
+                          sorted_strategies[i], "' on postgres database (",
+                          err, ")")
+              else
+                log(WARN, "unable to clean expired rows from table '",
+                          sorted_strategies[i], "' on postgres database")
+              end
             end
           end
+
+        else
+          log(ERR, "unable to clean expired rows from database (", err, ")")
```

Comment:
Would you mind unifying the above three error logs so that they all mention the "PostgreSQL database" for explicitness? See this existing log: https://github.com/Kong/kong/blob/1c2110cbc420c187fc76c3b7ada9846c8b3d762a/kong/db/strategies/postgres/connector.lua#L1106